Assets/Scripts/Frog.cs

```diff
@@ -4,12 +4,13 @@
 
 public class Frog : MonoBehaviour
 {
-    public float jumpTime;
+    // Let frog's jump time as a function of speed be f(spd) = 1/(spd^2).
```

Comment:
Formula feels a bit random... but after testing I guess we can stick with this bah, and tweak the difficulty accordingly (I was thinking about the physics equivalent of `t = s/v`, which did not have square).

Reply:
Ohh i randomly put this for diminishing decrease in jump time, otherwise the jump time becomes 0 very quickly. You got any other formula?

Reply:
Er but `1/(spd^2)` diminishes faster than `1/spd` though... But I guess since you guys already try 1.05x then it is ok bah.